flake.nix

```diff
@@ -111,6 +111,15 @@
           plutus-playground-server = topLevel.plutus-playground.server;
           marlowe-website = topLevel.marlowe-web;
           web-ghc-server = plutus.haskell.project.hsPkgs.web-ghc.components.exes.web-ghc-server;
+          # must be in $PATH of web-ghc-server
+          ghcWithPlutus = plutus.haskell.project.ghcWithPackages (ps: [
```

Comment:
Shouldn't we just expose a wrapped `web-ghc`? We have such a thing, it's here: https://github.com/input-output-hk/plutus/blob/master/nix/pkgs/default.nix#L164

Reply:
@michaelpj I think we need this for marlowe-playground as well, which currently doesn't use webghc. I'll update the comment and make a note to switch to the wrapped webghc.

Reply:
Well, we have some nixos modules lying around for the old deployment that AFAIK do this stuff right, so we should be able to borrow from that. That's where I found this web-ghc expression!

Reply:
Will be fixed vy https://github.com/input-output-hk/plutus/pull/3739